src/cart/crt_register.c

```diff
@@ -406,20 +404,16 @@ crt_opc_reg(struct crt_opc_info *opc_info, crt_opcode_t opc, uint32_t flags,
 		opc_info->coi_input_size;
 
 	/* set RPC features */
-	disable_reply =  flags & CRT_RPC_FEAT_NO_REPLY;
-	enable_reset_timer = flags & CRT_RPC_FEAT_NO_TIMEOUT;
-
-	opc_info->coi_no_reply = disable_reply;
-	if (disable_reply)
-		D_DEBUG(DB_TRACE, "opc %#x, reply disabled.\n", opc);
-	else
-		D_DEBUG(DB_TRACE, "opc %#x, reply enabled.\n", opc);
-
-	opc_info->coi_reset_timer = enable_reset_timer;
-	if (enable_reset_timer)
-		D_DEBUG(DB_TRACE, "opc %#x, reset_timer enabled.\n", opc);
-	else
-		D_DEBUG(DB_TRACE, "opc %#x, reset_timer disabled.\n", opc);
+	opc_info->coi_no_reply = (flags & CRT_RPC_FEAT_NO_REPLY) ? 1 : 0;
```

Comment:
Is there not a checkFlag macro or similar?

Reply:
not that i know of. i thought about adding one but seemed not worth for 3 cases

Reply:
#define set_bit_field(field, value) \
     (field) = (value) ? 1 : 0)

A generic macro would avoid the problem you mentioned yesterday of someone setting one in the future and not realizing that the flag actually has to be 1 rather than some random bit

Reply:
yeah i will add it